test/e2e/network/conntrack.go

```diff
@@ -442,6 +446,37 @@ var _ = common.SIGDescribe("Conntrack", func() {
 	// overcome this.
 	// xref: https://kubernetes.io/blog/2019/03/29/kube-proxy-subtleties-debugging-an-intermittent-connection-reset/
 	ginkgo.It("proxy implementation should not be vulnerable to the invalid conntrack state bug [Privileged]", func(ctx context.Context) {
+		// name of the target metric
+		metricName := "kubeproxy_iptables_ct_state_invalid_dropped_packets"
```

Comment:
so , the tests are meant to be agnostic of the implementation of the Services proxy, so you'll have to take one of these tests and replicate it in `test/e2e/network/kube_proxy.go`